### DISCUSSION TITLE
New Bank in spain

### DISCUSSION THREAD

**Discussion by javieralfonso80** [mdomke/schwifty#175] [Ideas]
in the bank list, there is a missing new bank for Spain
![image](https://github.com/mdomke/schwifty/assets/44975018/07623408-f6c7-4d0c-bd9d-f32ddeedbccd)

**Comment by mdomke** [comment]
The Revolut bank for Spain has been added in [2024.04.04](https://schwifty.readthedocs.io/en/latest/changelog.html#id6)
